AK/Statistics.h

```diff
@@ -51,16 +51,17 @@ class Statistics {
         return maximum;
     }
 
-    // FIXME: Implement a better algorithm
     T const median()
     {
         quick_sort(m_values);
```

Comment:
Could it be that you forgot to delete this line?

Reply:
No, it's deleted in a later commit (72a0face671fb505538dfd83346f41c2d4efd9a0) which is also in this pr where the algorithm is actually changed.